expath-pkg.xml.orig

```diff
@@ -0,0 +1,8 @@
+<<<<<<< HEAD
+<package xmlns="http://expath.org/ns/pkg" name="http://srophe.org/usaybia" abbrev="usaybia" version="0.7.1-dev" spec="1.0">
+=======
+<package xmlns="http://expath.org/ns/pkg" name="http://srophe.org/usaybia" abbrev="usaybia" version="0.5.1" spec="1.0">
+>>>>>>> master
+    <title>Usaybia.net</title>
+    <dependency package="http://exist-db.org/apps/shared"/>
+</package>
```

Comment:
@wsalesky I think this .orig file is just an artifact of the conflict resolution? I'll delete it.